README.md

```diff
@@ -44,8 +44,8 @@ On this same page, select `JavaScript` at the bottom to download your `aws-expor
 Start the application:
 
 ```
-yarn
-yarn start
+npm
+npm run start
```

Comment:
if we are switching to npm, can you remove the `yarn.lock`?

Reply:
Good catch, thanks!